R/spatial-functions.R

```diff
@@ -176,3 +176,34 @@ covar_exppow <- function(d, range, shape){
   exp(-(d/range)^shape)
 }
 
+
+#' calculate maximum distance among a table of coordinates
+#'
+#' @param coords the coordinate matrix (or dataframe) from which a maximum distance is desired.
+#' @param dist_FUN the distance function used to calculate distances
+#'
+#' @details First the outermost points are found by fitting a convex hull in
+#' Euclidean space. Then, the distances between these outer points is calculated
```

Comment:
This does mean that we could be missing the **true** boundary positions (because of assuming a flat plane instead of spheroid projections of the coordinates). But I think it falls under close enough.

What do you think, @arives and @erfea?

Reply:
See #17 for further side-effects. Perhaps this is a bad solution entirely.

Reply:
I'm not sure what this is going to do with a global dataset (on a sphere); Likai is analyzing a global dataset right now. Sorry, but I think it is better to just go with how it was before, just using the maximum in the partitioned sample.

Reply:
Fair enough!